core/src/test/java/com/pholser/junit/quickcheck/runner/PropertyFalsifiedUtilityClassTest.java

```diff
@@ -26,11 +26,119 @@ a copy of this software and associated documentation files (the
+        final AssertionError originalFailure = new AssertionError(assertionName);
+        final AssertionError actual = PropertyFalsified.smallerCounterexampleFound(
+            propertyName,
+            originalArguments,
+            arguments,
+            seeds,
+            smallerFailure,
+            originalFailure);
+        final String expected = "Property named 'mySuperProperty' failed (smaller name):\n"
+            + "With arguments: [first]\n"
+            + "Original failure message: assertion name\n"
+            + "First arguments found to also provoke a failure: [first, second, third]\n"
+            + "Seeds for reproduction: [12345, 8842]";
+        Assert.assertEquals(expected, actual.getMessage());
+    }
+
+    @Test
+    public void smallerCounterexampleFoundIsCorrectEvenIfSmallerFailureIsNotNamed() {
+        final String propertyName = "mySuperProperty";
+        final String[] originalArguments = {"first", "second", "third"};
+        final String[] arguments = {"first"};
+        final long[] seeds = {12345, 8842};
+        final AssertionError smallerFailure = new AssertionError();
+        final String assertionName = "assertion name";
+        final AssertionError originalFailure = new AssertionError(assertionName);
+        final AssertionError actual = PropertyFalsified.smallerCounterexampleFound(
+            propertyName,
+            originalArguments,
+            arguments,
+            seeds,
+            smallerFailure,
+            originalFailure);
+        final String expected = "Property named 'mySuperProperty' failed:\n"
+            + "With arguments: [first]\n"
+            + "Original failure message: assertion name\n"
+            + "First arguments found to also provoke a failure: [first, second, third]\n"
+            + "Seeds for reproduction: [12345, 8842]";
+        Assert.assertEquals(expected, actual.getMessage());
+    }
+
+    @Test
+    public void smallerCounterexampleFoundIsCorrectEvenIfOriginalFailureIsNotNamed() {
+        final String propertyName = "mySuperProperty";
+        final String[] originalArguments = {"first", "second", "third"};
+        final String[] arguments = {"first"};
+        final long[] seeds = {12345, 8842};
+        final AssertionError smallerFailure = new AssertionError();
+        final AssertionError originalFailure = new AssertionError();
+        final AssertionError actual = PropertyFalsified.smallerCounterexampleFound(
+            propertyName,
+            originalArguments,
+            arguments,
+            seeds,
+            smallerFailure,
+            originalFailure);
+        final String expected = "Property named 'mySuperProperty' failed:\n"
+            + "With arguments: [first]\n"
+            + "First arguments found to also provoke a failure: [first, second, third]\n"
+            + "Seeds for reproduction: [12345, 8842]";
+        Assert.assertEquals(expected, actual.getMessage());
+    }
```

Comment:
Can you modify src/site/markdown/usage/seed.md to reflect the new format of the failure message?